gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -744,6 +745,8 @@
    * Hive registration properties
    */
   public static final String HIVE_REGISTRATION_POLICY = "hive.registration.policy";
+  // Set to true if hive registration is done in taskStateTracker on task commit.
+  public static final String PIPELINED_HIVE_REGISTRATION_ENABLE = "hive.registration.pipelined.enabled";
```

Comment:
Since the handler is not necessary to do hive registration, does it make sense to keep this conf. for specifying the handler is actually handling hive registration ?